Source/Charts/Utils/Platform.swift

```diff
@@ -39,11 +39,6 @@ extension UIScreen
     }
 }
 
-func NSUIMainScreen() -> NSUIScreen?
```

Comment:
@jjatie I found this func is duplicated under two macros iOS & macOS; I manually tested on my iOS simulator and macOS, it seems we only need one outside these macros. We are safe to delete one right? I don't come up a case that we need to put them into two places

Reply:
Yea, we don't need both. Really this should be an extension on `NSUIScreen`
```
extension NSUIScreen {
    var nsuiMain: Self? { .main }
}
```

Reply:
@jjatie I have pushed a new commit. but mine is
```
class var nsuiMain: NSUIScreen? { .main }
```
is there any reason to use `Self?` here? Because if I use `Self?`, Xcode would complain
>Type 'Self?' has no member 'main'

and I have to use something like
```
class var nsuiMain: Self? { NSUIScreen.main as? Self }
```
(I can't even use `{ .main as? Self }` ) So I think use NSUIScreen explicitly makes it cleaner.